test/MagicString.js

```diff
@@ -1305,7 +1305,28 @@ describe('MagicString', () => {
 
 			s.replace('2', '3');
 
-			assert.strictEqual(s.toString(), '1 3 1 2');
+			assert.strictEqual(s.toString(), '1 3 1 3');
```

Comment:
If we don't want add `.replaceAll`, I guess string replace should replace them all.

Reply:
I guess it makes better sense to align with JavaScript's string, so I'd prefer adding `.replaceAll` and keeping `.replace` only replaced once.

Reply:
Sure!